base/loading.jl

```diff
@@ -712,7 +712,7 @@ function project_file_manifest_path(project_file::String)::Union{Nothing,String}
     cache = LOADING_CACHE[]
     if cache !== nothing
         manifest_path = get(cache.project_file_manifest_path, project_file, missing)
-        manifest_path === missing || return manifest_path
+        ismissing(manifest_path) || return manifest_path
```

Comment:
Here it is literally `missing` on the line above so it will never be something missing that is not `missing`. What is the reason for a change like this (and the one in the same file above; it is not like `manifest_uuid_path` will suddenly return missing values that is not `missing`, or?). Personally I find `path === missing` easier to read compared to `ismissing(path)`.

Reply:
It's true that this is not needed here. I applied a regex search and replace.

Anyways, this PR is a duplicate.